pages/docs/couple-your-code/couple-your-code-moving-or-changing-meshes.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Moving or changing meshes
+permalink: couple-your-code-moving-or-changing-meshes.html
+keywords: api, adapter, ALE, geometries, CFD
+summary: "preCICE supports ALE-methods, direct-mesh-access, and pseudo-meshes to handle most scenarios of moving or changing meshes. A remeshing API is on the roadmap and being actively developped."
+---
+
+## Static reference domain
+
+preCICE supports static meshes, meaning dynamic meshes can still be emulated using node displacements as described by the Arbitrary-Lagrangian-Eulerian or ALE-methods.
+
+Note that spatially aware mapping-schemes in preCICE work in the reference domain, not the physical domain.
+
+Examples of this method are the [perpendicular flap tutorials](tutorials-perpendicular-flap) and the [Turek-Hron FSI3 benchmark](tutorials-turek-hron-fsi3).
+
+## Mapping inside adapter
+
+In some cases, the mesh of a participant may fluctuate extremely between timestep or it may be unclear, where exactly sampling points are needed for following timesteps.
```

Comment:
```suggestion
In some cases, the mesh of a participant may fluctuate extremely between time steps or it may be unclear, where exactly sampling points are needed for following time steps.
```

At some point, we fixed that we use "time step" not "timestep".